cmd/podman/common/create.go

```diff
@@ -697,6 +697,14 @@ func DefineCreateFlags(cmd *cobra.Command, cf *ContainerCLIOpts) {
 	)
 	_ = cmd.RegisterFlagCompletionFunc(systemdFlagName, AutocompleteSystemdFlag)
 
+	timeoutFlagName := "timeout"
+	createFlags.UintVar(
+		&cf.Timeout,
+		timeoutFlagName, 0,
+		"Timeout (in seconds) to kill a container if it does not complete, by default container will not be stopped",
```

Comment:
Instead of "by default..." - "A timeout of 0 means the container will not be automatically stopped"

Reply:
Maybe should reword the first bit, too - "Maximum length of time a container can run. They will be stopped automatically afterwards.

Reply:
Fixed